docs/2_5_fmu_distribution.adoc

```diff
@@ -95,6 +95,9 @@ Each <<BuildConfiguration>> provides the necessary information to compile and li
 An FMU importing tool may not regard more than one <<BuildConfiguration>> when building the FMU for a specific <<platform>>.
 The importer chooses the matching <<BuildConfiguration>> based on the <<platform>> and <<modelIdentifier>> attributes.
 
+If only a single <<SourceFile>> is provided in the <<BuildConfiguration>> it should have a unique name (e.g. `<model_identifier>.c`) and all symbols except for the FMI functions should be defined as `static` to avoid name conflicts when compiling and linking multiple source code FMUs into one binary.
```

Comment:
Can you clarify what `unique name` refers to? How can an exporting tool ensure that its only source file name is unique w.r.t. to the importing environment?

Reply:
Maybe we should better call it „non-generic“ (like model.c or all.c)

Reply:
Thus, it is not about avoiding name conflicts, but just to minimize the danger of them, right?

Reply:
We could not ensure this - source files are customer files we bundle and we cannot even wrap them nicely (using #include) as suggested without getting into name-conflict trouble. We should rather suggest to ensure each is compiled in their own subdirectory to avoid file-name clashes.
Note: Even then, merging sources from different source-code FMUs may create conflicts: global variables and function names, even types can have the same name but point to unique objects.
We cannot solve this perfectly, but we can improve the likelihood of success with some suggestions (again, "rules" are too strong, packaging customer code into FMUs...).

Reply:
> Thus, it is not about avoiding name conflicts, but just to minimize the danger of them, right?

Yes. It's basically applying the function prefix concept from the headers (`<model_identifier>_`) to the filename (and hence the object file).